_clients/go.md

```diff
@@ -11,21 +11,171 @@ The OpenSearch Go client lets you connect your Go application with the data in y
 
 ## Setup
 
-If you're creating a new project:
+If you're starting a new project, create a new module by running:
 
 ```go
-go mod init
+go mod init <mymodulename>
 ```
 
-To add the client to your project, import it like any other module:
+To add the Go client to your project, import it like any other module:
 
 ```go
 go get github.com/opensearch-project/opensearch-go
 ```
+## Connecting to OpenSearch
 
-## Sample code
+To connect to the default OpenSearch host, create a client object as follows:
```

Comment:
Global: Instead of "as follows", which could be a bit ambiguous, can we rephrase these introductory sentences to be a little more clear? Something like "...create a client object by running the following code" or "...use the following to create...". Please apply to all instances of "as follows".